components/GeoApi.js

```diff
@@ -0,0 +1,77 @@
+import React, { useEffect, useState } from "react";
+import { resolve } from "styled-jsx/css";
+import style from "../styles/Home.module.css";
+
+function GeoApi(props) {
+  const { weatherForecast, setWeatherForecast } = props;
+  const [locationName, setLocationName] = useState("Linn, Kansas");
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const handleChange = (e) => {
+    setSearchTerm(e.target.value);
+  };
+
+  async function fetchCoords() {
+    let long = 0;
+    let lat = 0;
+    let firstRes = await fetch(`http://api.positionstack.com/v1/forward?access_key=${process.env.NEXT_PUBLIC_GEOLOCATION_API_KEY}&query=${searchTerm}`);
+    let firstData = await firstRes.json();
+    let a = 0;
+    while (firstData.data[a].continent !== "North America") {
+      a++;
+    }
+    long = firstData.data[a].longitude;
+    lat = firstData.data[a].latitude;
+    console.log(a, firstData);
+  }
+
+  async function fetchWeatherForecast() {
```

Comment:
I like the separate functions wrapping the API calls here! Helps me understand what the purpose of the API call is.